html-forms/index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>HTML Forms</title>
+</head>
+<body>
+  <h1>Coding Form!</h1>
+  <form>
+    <div>
+      <label for="email">Email</label>
+      <br>
+      <input type="text" id="email">
+    </div>
+    <div>
+      <label for="coding">Why is coding cool?</label>
+    </div>
+    <select id="coding">
+      <option value="option">Select an option</option>
+      <option value="interesting">Because it's interesting</option>
+      <option value="fun">Because it's fun</option>
+      <option value="powerful">Because it's powerful</option>
+    </select>
+    <p>Favorite Language (choose one)</p>
+    <input type="radio" id="html" name="answer" value="HTML">
+    <label for="html">HTML</label>
+    <input type="radio" id="css" name="answer" value="CSS">
```

Comment:
This element's id is not the same case as the value of the `for` attribute on its label, make sure those match!